.github/jobs/SimpleGradleJob.pkl

```diff
@@ -0,0 +1,20 @@
+extends "GradleJob.pkl"
+
+name: String = command
+
+command: String
+
+os = "linux"
+
+javaVersion = "21.0"
+
+steps {
+  new {
+    name = module.name
+    shell = "bash"
```

Comment:
Do we specifically care about overriding the default "unspecified" behavior here? https://docs.github.com/en/actions/reference/workflows-and-actions/workflow-syntax#jobsjob_idstepsshell